protected/humhub/modules/tour/TourConfig.php

```diff
@@ -0,0 +1,205 @@
+    public const TOUR_ID_ADMINISTRATION = 'administration';
+    /**
+     * string
+     * A unique ID of the Tour
+     */
+    public const KEY_TOUR_ID = 'tour_id';
+    /**
+     * bool
+     * Optional. Set to false for hiding the Tour from the list
+     */
+    public const KEY_IS_VISIBLE = 'is_visible';
+    /**
+     * string
+     * The title of the Tour displayed in the widget
+     */
+    public const KEY_TITLE = 'title';
+    /**
+     * string|null
+     * Optional The Controller class of the Tour where it must be started
+     * If null or not defined, the KEY_START_URL will be used instead
+     */
+    public const KEY_TOUR_ON_CONTROLLER_CLASS = 'tour_on_controller_class';
+    /**
+     * string
+     * Starting URL of the Tour
+     */
+    public const KEY_START_URL = 'start_url';
+    /**
+     * string|null
+     * ID of the next Tour to show when the current the Tour is finished
+     */
+    public const KEY_NEXT_TOUR_ID = 'next_tour_id';
+    /**
+     * array
+     * Driver.js configuration and steps
+     * Available values: https://driverjs.com/docs/
+     */
+    public const KEY_DRIVER_JS = 'driver_js';
+
+    /**
+     * Return an array of valid and visible Tour configs
+     * The array keys are the Tour IDs
+     */
+    public static function get(): array
+    {
+        /* @var Module $module */
+        $module = Yii::$app->getModule('tour');
+
+        $tourConfigs = [];
+        foreach ($module->tourConfigFiles as $file) {
+            $config = require Yii::getAlias($file);
+            if (static::isValidConfig($config) && static::getIsVisible($config)) {
+                $tourConfigs[$config[self::KEY_TOUR_ID]] = $config;
+            }
+        }
+
+        return $tourConfigs;
+    }
+
+    public static function isValidConfig($config): bool
+    {
+        $isValid =
+            is_array($config)
+            && !empty($config[self::KEY_TOUR_ID])
+            && !empty($config[self::KEY_TITLE])
+            && !empty($config[self::KEY_START_URL])
+            && array_key_exists(self::KEY_NEXT_TOUR_ID, $config)
+            && !empty($config[self::KEY_DRIVER_JS]);
+
+        if (!$isValid) {
+            Yii::error("Invalid Tour params: " . print_r($config, true), 'tour');
+        }
+
+        return $isValid;
+    }
+
+    public static function getCurrent(): ?array
+    {
+        foreach (self::get() as $config) {
+            $requiredControllerClass = static::getRequiredControllerClass($config);
+            if (
+                ($requiredControllerClass && Yii::$app->controller instanceof $requiredControllerClass)
+                || Url::current() === static::getStartUrl($config)
+            ) {
+                return $config;
+            }
+        }
+
+        return null;
+    }
+
+    public static function IsCurrentRouteAcceptable(string $tourId): bool
+    {
+        return array_key_exists($tourId, self::get());
+    }
+
+
+    public static function getTourSpace(): ?Space
```

Comment:
Why this not as callable/anon func? So we have no tour related stuff in this class?

Maybe we could move this in helpers\SpaceTourHelper.php

Reply:
Sure. Forgot about it when moving config to dedicated files.
Done in commit https://github.com/humhub/humhub/pull/7535/commits/d9cc587f630a199ae4480b522c7f25a02d8c1cb1